ui/widgets/sidebar/src/components/sidebar-component.tsx

```diff
@@ -151,7 +151,7 @@ const SidebarComponent: React.FC<RootComponentProps> = props => {
     <>
       <SidebarOverlay onClick={handleSidebarClose} />
       <AppSidebar
-        versionLabel="ALPHA"
+        versionLabel="TEST"
```

Comment:
the text `TEST` or `DEV` should be displayed only there is `__DEV__` env value set

Reply:
Thanks @kenshyx. I have fixed it.